std/assembly/rt/tlsf.ts

```diff
@@ -428,10 +432,8 @@ function growMemory(root: Root, size: usize): void {
     return;
   }
   // Here, both rounding performed in searchBlock ...
-  const halfMaxSize = BLOCK_MAXSIZE >> 1;
-  if (size < halfMaxSize) { // don't round last fl
-    const invRound = (sizeof<usize>() * 8 - 1) - SL_BITS;
-    size += (1 << (invRound - clz<usize>(size))) - 1;
+  if (size > SB_SIZE) { // don't round last fl
```

Comment:
Should this be `>= SB_SIZE`, given that in `searchBlock`, it's `< SB_SIZE`?

Reply:
And it looks like the trailing comment regarding rounding can be removed, as it now lives in the refactored function